generators/entity-server/templates/dotnetcore/src/Project.Infrastructure/Data/ApplicationDatabaseContext.cs.ejs

```diff
@@ -75,6 +84,16 @@ namespace <%= namespace %>.Infrastructure.Data {
         {
             base.OnModelCreating(builder);
             <%_ if (authenticationType === 'jwt' && applicationType !== 'microservice') { _%>
+
+            // Rename AspNet default tables
+            builder.Entity<User>().ToTable("Users");
```

Comment:
Why rename default name ?

Reply:
to delete "AspNet" prefix for generated Identity tables. I also noticed that the generated tables are not used except Users/Roles/UserRoles